src/test/java/dog/pawbook/storage/JsonAddressBookStorageTest.java

```diff
@@ -46,34 +46,34 @@ private Path addToTestDataPathIfNotNull(String prefsFileInTestDataFolder) {
 
     @Test
     public void read_missingFile_emptyResult() throws Exception {
-        assertFalse(readAddressBook("NonExistentFile.json").isPresent());
+        assertFalse(readDatabase("NonExistentFile.json").isPresent());
     }
 
     @Test
     public void read_notJsonFormat_exceptionThrown() {
-        assertThrows(DataConversionException.class, () -> readAddressBook("notJsonFormatAddressBook.json"));
+        assertThrows(DataConversionException.class, () -> readDatabase("notJsonFormatAddressBook.json"));
```

Comment:
do you want to also rename the json files?

Reply:
Edited.